### DISCUSSION TITLE
Unable to launch

### DISCUSSION THREAD

**Discussion by mark-bakermd** [EricAndrechek/homebridge-display#162] [Q&A]
I have tried providing multiple port numbers including 8080 and 6969 however whenever I reboot homebridge and attempt to access the homebridge-display gui using the specified port number I always get This site can’t be reached / homebridge.mydomain.com refused to connect.

Are there logs I can check to figure out why this app is not making its server binding?

**Comment by dalejake** [comment]
same here , tried multiple devises and browsesrs on the same wifi network
